src/main/resources/db/migration/V1_16__update_prison_names.sql

```diff
@@ -0,0 +1,67 @@
+UPDATE PRISON SET NAME = 'AYLESBURY (HMYOI)' WHERE PRISON_ID = 'AYI';
+UPDATE PRISON SET NAME = 'BEDFORD (HMP & YOI)' WHERE PRISON_ID = 'BFI';
+UPDATE PRISON SET NAME = 'BELMARSH (HMP & YOI)' WHERE PRISON_ID = 'BAI';
+UPDATE PRISON SET NAME = 'BERWYN (HMP & YOI)' WHERE PRISON_ID = 'BWI';
+UPDATE PRISON SET NAME = 'BRINSFORD (HMP & YOI)' WHERE PRISON_ID = 'BSI';
+UPDATE PRISON SET NAME = 'BRISTOL (HMP & YOI)' WHERE PRISON_ID = 'BLI';
+UPDATE PRISON SET NAME = 'BRONZEFIELD (HMP & YOI)' WHERE PRISON_ID = 'BZI';
+UPDATE PRISON SET NAME = 'BULLINGDON (HMP & YOI)' WHERE PRISON_ID = 'BNI';
+UPDATE PRISON SET NAME = 'CARDIFF (HMP & YOI)' WHERE PRISON_ID = 'CFI';
+UPDATE PRISON SET NAME = 'CHELMSFORD (HMP & YOI)' WHERE PRISON_ID = 'CDI';
+UPDATE PRISON SET NAME = 'CHELMSFORD (HMP & YOI)' WHERE PRISON_ID = 'CDI';
+UPDATE PRISON SET NAME = 'COOKHAM WOOD (HMYOI)' WHERE PRISON_ID = 'CKI';
+UPDATE PRISON SET NAME = 'DEERBOLT (HMPYOI)' WHERE PRISON_ID = 'DTI';
+UPDATE PRISON SET NAME = 'DONCASTER (HMP & YOI)' WHERE PRISON_ID = 'DNI';
+UPDATE PRISON SET NAME = 'DOWNVIEW (HMP & YOI)' WHERE PRISON_ID = 'DWI';
+UPDATE PRISON SET NAME = 'DRAKE HALL (HMP & YOI)' WHERE PRISON_ID = 'DHI';
+UPDATE PRISON SET NAME = 'DURHAM (HMP & YOI)' WHERE PRISON_ID = 'DMI';
+UPDATE PRISON SET NAME = 'EAST SUTTON PARK (HMP & YOI)' WHERE PRISON_ID = 'ESI';
+UPDATE PRISON SET NAME = 'EASTWOOD PARK (HMP & YOI)' WHERE PRISON_ID = 'EWI';
+UPDATE PRISON SET NAME = 'ELMLEY (HMP & YOI)' WHERE PRISON_ID = 'EYI';
+UPDATE PRISON SET NAME = 'ERLESTOKE (HMP & YOI)' WHERE PRISON_ID = 'EEI';
+UPDATE PRISON SET NAME = 'EXETER (HMP & YOI)' WHERE PRISON_ID = 'EXI';
+UPDATE PRISON SET NAME = 'FOREST BANK (HMP & YOI)' WHERE PRISON_ID = 'FBI';
+UPDATE PRISON SET NAME = 'FOSTON HALL (HMP & YOI)' WHERE PRISON_ID = 'FHI';
+UPDATE PRISON SET NAME = 'HATFIELD (HMP & YOI)' WHERE PRISON_ID = 'HDI';
+UPDATE PRISON SET NAME = 'HIGH DOWN (HMP & YOI)' WHERE PRISON_ID = 'HOI';
+UPDATE PRISON SET NAME = 'HINDLEY (HMP & YOI)' WHERE PRISON_ID = 'HII';
+UPDATE PRISON SET NAME = 'HOLLESLEY BAY (HMP & YOI)' WHERE PRISON_ID = 'HBI';
+UPDATE PRISON SET NAME = 'HOLME HOUSE (HMP & YOI)' WHERE PRISON_ID = 'HHI';
+UPDATE PRISON SET NAME = 'HULL (HMP & YOI)' WHERE PRISON_ID = 'HLI';
+UPDATE PRISON SET NAME = 'ISLE OF WIGHT (HMP & YOI)' WHERE PRISON_ID = 'IWI';
+UPDATE PRISON SET NAME = 'KIRKLEVINGTON GRANGE (HMP & YOI)' WHERE PRISON_ID = 'KVI';
+UPDATE PRISON SET NAME = 'LEWES (HMP & YOI)' WHERE PRISON_ID = 'LWI';
+UPDATE PRISON SET NAME = 'LINCOLN (HMP & YOI)' WHERE PRISON_ID = 'LII';
+UPDATE PRISON SET NAME = 'LOW NEWTON (HMP & YOI)' WHERE PRISON_ID = 'LNI';
+UPDATE PRISON SET NAME = 'MANCHESTER (HMP & YOI)' WHERE PRISON_ID = 'MRI';
+UPDATE PRISON SET NAME = 'MOORLAND (HMP & YOI)' WHERE PRISON_ID = 'MDI';
+UPDATE PRISON SET NAME = 'MORTON HALL
+(Opening as an Adult Male prison Late 2021)' WHERE PRISON_ID = 'MHI';
+UPDATE PRISON SET NAME = 'NORWICH (HMP & YOI)' WHERE PRISON_ID = 'NWI';
```

Comment:
I don't think we need the text about when it's opening. - Should be Morton Hall (HMP)